### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by arminvoid**

### CODE REVIEW

File: contracts/Optimist.sol [hunk 1]
```diff
@@ -1,6 +0,0 @@
-pragma solidity ^0.4.25;
-
-interface Optimist {
```
Comment (izqui) [inline contracts/Optimist.sol]: Why was this interface removed?

**Review by izqui — APPROVED** [review]
Looking really good!

File: contracts/Optimist.sol (hunk 1 above)
Comment (atvanguard) [reply contracts/Optimist.sol]: I started with writing that interface but didn't come around inheriting from it. In an interface all declared functions must be external, whereas I wanted to have internal functions in `mintNotes`, `spendNotes` and `liquidateNotes`.

Comment (izqui) [reply contracts/Optimist.sol]: Sounds good

**Pull request merged**